projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.scss

```diff
@@ -25,6 +25,22 @@
   z-index: z-index(off-canvas);
 }
 
+.go-off-canvas.large {
+  width: 75vw;
+  background-color: $theme-light-app-bg;
+  color: $base-dark;
+  .go-off-canvas__header {
+    background-color: $base-light;
```

Comment:
Should we use the `$theme-dark-color` variable here instead of `$base-light`?

Reply:
Did you mean `$theme-light-bg`?

Reply:
Yeah, my bad.